spec/index.bs

```diff
@@ -634,6 +634,7 @@ Each such <dfn dfn>texture subresource</dfn> is uniquely identified by a
 
 We define <dfn dfn>subresource</dfn> to be either a whole buffer, or a [=texture subresource=].
 
+<div algorithm>
 Some [=internal usage=]s are compatible with others. A [=subresource=] can be in a state
 that combines multiple usages together. We consider a list |U| to be 
 a <dfn dfn>compatible usage list</dfn> if (and only if) it satisfies any of the following rules:
```

Comment:
I think this makes the algorithm title "compatible usage list", which is fine, just fyi